.github/workflows/main.yml

```
@@ -0,0 +1,114 @@
+      - main
+    paths:
+      - 'API/**'
+      - 'client/**'
+      - '.github/workflows/main.yml'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)